docs/documentation/helm.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Helm
+description: Helm
+layout: docs
+permalink: /docs/helm
+---
+
+### Helm
+
+Dekorate also supports generating manifests for `helm`. To make use of this feature you need to add:
+
+```xml
+<dependency>
+  <groupId>io.dekorate</groupId>
+  <artifactId>helm-annotations</artifactId>
+  <version>{{site.data.project.release.current-version}}</version>
+</dependency>
+```
+
+This module provides the [@HelmChart](annotations/helm-annotations/src/main/java/io/dekorate/helm/annotation/HelmChart.java) that will generate the following Helm resources:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+- <chart name>-<chart version>-helm.tar.gz
+
+#### Getting Started
+
+Add your Helm chart configuration in your properties like:
```

Comment:
Please better introduce this getting started section as we cannot start with `Add your Helm chart ...`

Reply:
Changed